components/pipeline-service/kustomization.yaml

```diff
@@ -2,14 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- openshift-pipelines
-- pipelines-as-code
-- pvc-cleaner
-- tekton-chains
-- tekton-results
-
-# Skip applying the Tekton operands while the Tekton operator is being installed.
-# See more information about this option, here:
-# https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#skip-dry-run-for-new-custom-resources-types
-commonAnnotations:
-  argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+- git::https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=new-main
```

Comment:
Also not sure why you are not using official release and just patch that in pipeline-service.

Reply:
> Not sure why user ID is enforced, normally openshift selects random user by default.

I asked the same question to @ramessesii2 here https://github.com/openshift-pipelines/pipeline-service/pull/376#discussion_r1026147897 and my understanding is that he followed the upstream tekton config.

> Also not sure why you are not using official release and just patch that in pipeline-service.

@avinal is working on that he probable has an answer

Reply:
> my understanding is that he followed the upstream tekton config.

I was following this [Deployment](https://github.com/tektoncd/pipeline/blob/main/config/controller.yaml#L132) from tekton-pipelines controller. 

> Invalid value: 65532: must be in the ranges: [1000940000, 1000949999], provider "restricted": Forbidden: not usable by user or serviceaccount, provider "nonroot-v2": Forbidden: not usable by user or serviceaccount, provider "nonroot"

I'm not sure why defining a UID gives this error.

Reply:
@ramessesii2, in openshift, using a specific UID seems to require a particular SCC to be defined first.
Anyway the link you provided is about tekton-controller, maybe it is required there in order to correctly handle permission in mounted volumes. That does not seem to be required in tekton-results or chains.
So I think we can proceed with removing that part and still filling the security requirements of your PR https://github.com/openshift-pipelines/pipeline-service/pull/376

Reply:
I agree 👍🏼

Reply:
I have removed `runAsUser` field

Reply:
> Also not sure why you are not using official release and just patch that in pipeline-service. 

@Mo3m3n @Michkov There have been new additions to the tekton results since the last release. I have already updated our fork. Will need to update the images in the pipeline service.